Leftwm won't build (on my machine)

### Describe the bug

Building leftwm through `cargo install leftwm` results in a compilation error.
Looks like the problem is with `lefthk`.

I've managed to build it with `--no-default-features`, but I would like to have `lefthk` (adding `--features lefthk` errors with the same message).

### Expected behavior (optional)

_No response_

### Steps to reproduce (optional)

_No response_

### Relevant log output (optional)

```text
❯ cargo install leftwm
    Updating crates.io index
  Installing leftwm v0.4.2
    Updating crates.io index
   Compiling proc-macro2 v1.0.69
   Compiling unicode-ident v1.0.12
   Compiling libc v0.2.149
   Compiling autocfg v1.1.0
   Compiling serde v1.0.190
   Compiling thiserror v1.0.50
   Compiling memchr v2.6.4
   Compiling once_cell v1.18.0
   Compiling pin-project-lite v0.2.13
   Compiling cfg-if v1.0.0
   Compiling futures-core v0.3.29
   Compiling log v0.4.20
   Compiling itoa v1.0.9
   Compiling futures-task v0.3.29
   Compiling regex-syntax v0.8.2
   Compiling ucd-trie v0.1.6
   Compiling futures-channel v0.3.29
   Compiling pin-utils v0.1.0
   Compiling pkg-config v0.3.27
   Compiling futures-sink v0.3.29
   Compiling futures-util v0.3.29
   Compiling tracing-core v0.1.32
   Compiling futures-io v0.3.29
   Compiling time-core v0.1.2
   Compiling slab v0.4.9
   Compiling memoffset v0.7.1
   Compiling num-traits v0.2.17
   Compiling signal-hook v0.3.17
   Compiling powerfmt v0.2.0
   Compiling time-macros v0.2.15
   Compiling either v1.9.0
   Compiling utf8parse v0.2.1
   Compiling aho-corasick v1.1.2
   Compiling proc-macro-hack v0.5.20+deprecated
   Compiling deranged v0.3.9
   Compiling bytes v1.5.0
   Compiling x11-dl v2.21.0
   Compiling num_threads v0.1.6
   Compiling syn v1.0.109
   Compiling quote v1.0.33
   Compiling static_assertions v1.1.0
   Compiling regex-syntax v0.6.29
   Compiling serde_json v1.0.108
   Compiling syn v2.0.38
   Compiling itertools v0.10.5
   Compiling anstyle-parse v0.2.2
   Compiling colorchoice v1.0.0
   Compiling ryu v1.0.15
   Compiling bitflags v1.3.2
   Compiling anstyle-query v1.0.0
   Compiling hashbrown v0.14.2
   Compiling overload v0.1.1
   Compiling anstyle v1.0.4
   Compiling xdg v2.5.2
   Compiling option-ext v0.2.0
   Compiling equivalent v1.0.1
   Compiling lazy_static v1.4.0
   Compiling doc-comment v0.3.3
   Compiling anymap2 v0.13.0
   Compiling anstream v0.6.4
   Compiling signal-hook-registry v1.4.1
   Compiling num_cpus v1.16.0
   Compiling socket2 v0.5.5
   Compiling mio v0.8.9
   Compiling regex-automata v0.4.3
   Compiling dirs-sys-next v0.1.2
   Compiling time v0.3.30
   Compiling regex-automata v0.1.10
   Compiling dirs-next v2.0.0
   Compiling indexmap v2.1.0
   Compiling dirs-sys v0.4.1
   Compiling nix v0.26.4
   Compiling sharded-slab v0.1.7
   Compiling nu-ansi-term v0.46.0
   Compiling matchers v0.1.0
   Compiling tracing-log v0.1.4
   Compiling thread_local v1.1.7
   Compiling percent-encoding v2.3.0
   Compiling unicode-xid v0.2.4
   Compiling unicode-segmentation v1.10.1
   Compiling winnow v0.5.19
   Compiling base64 v0.21.5
   Compiling strsim v0.10.0
   Compiling anyhow v1.0.75
   Compiling clap_lex v0.6.0
   Compiling regex v1.10.2
   Compiling smallvec v1.11.1
   Compiling clap_builder v4.4.7
   Compiling const_format_proc_macros v0.2.32
   Compiling dirs v5.0.1
   Compiling leftwm v0.4.2
   Compiling inventory v0.3.13
   Compiling shellexpand v3.1.0
   Compiling const_format v0.2.32
   Compiling git-version-macro v0.3.5
   Compiling clap v4.4.7
   Compiling git-version v0.3.5
   Compiling serde_derive v1.0.190
   Compiling thiserror-impl v1.0.50
   Compiling tracing-attributes v0.1.27
   Compiling futures-macro v0.3.29
   Compiling tokio-macros v2.1.0
   Compiling liquid-derive v0.26.4
   Compiling pest v2.7.5
   Compiling tokio v1.33.0
   Compiling tracing v0.1.40
   Compiling tracing-subscriber v0.3.17
   Compiling pest_meta v2.7.5
   Compiling pest_generator v2.7.5
   Compiling tracing-journald v0.3.0
   Compiling pest_derive v2.7.5
   Compiling futures-executor v0.3.29
   Compiling futures v0.3.29
   Compiling bitflags v2.4.1
   Compiling kstring v2.0.0
   Compiling serde_spanned v0.6.4
   Compiling toml_datetime v0.6.5
   Compiling toml_edit v0.19.15
   Compiling ron v0.8.1
   Compiling nix v0.27.1
   Compiling liquid-core v0.26.4
   Compiling leftwm-core v0.4.2
   Compiling lefthk-core v0.1.9
   Compiling liquid-lib v0.26.4
   Compiling toml v0.7.8
   Compiling liquid v0.26.4
   Compiling xlib-display-server v0.1.1
error[E0599]: no associated item named `Execute` found for trait `lefthk_core::config::Command`
   --> /home/nitzan/.cargo/registry/src/index.crates.io-6f17d22bba15001f/leftwm-0.4.2/src/config/keybind.rs:106:52
    |
106 |             command: lefthk_core::config::Command::Execute(command),
    |                                                    ^^^^^^^
    |                                                    |
    |                                                    associated item not found in `Command`
    |                                                    help: there is a method with a similar name: `execute`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `leftwm` (lib) due to previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `leftwm v0.4.2`, intermediate artifacts can be found at `/tmp/cargo-installDrU1bU`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```


### OS / Distro

Linux Mint 21.2 (Edge) (Cinnamon)

### Additional System Information (optional)

_No response_

### LeftWM Check

```text
❯ leftwm-check
:: LeftWM version: 0.4.2
:: LeftWM git hash: unknown
:: Built with no enabled features.
:: Checking for leftwm binaries . . .
    -> Binaries OK
:: Loading configuration . . .
    -> Configuration loaded OK
WARN: Ignoring checks on keybinds as you compiled for an external hot key daemon.
:: Checking environment . . .
    -> Environment OK
:: Checking theme . . .
WARN: TOML as config format is about to be deprecated.
      Please consider migrating to RON or contact the theme creator about this topic.
      Note: make sure the `up` script is loading the correct theme file.
ERROR: File not found: theme.ron
```

--- a/leftwm/Cargo.toml
+++ b/leftwm/Cargo.toml
@@ -19,7 +19,7 @@ once_cell = "1.13.0"
 dirs-next = "2.0.0"
 futures = "0.3.21"
 git-version = "0.3.5"
-lefthk-core = { version = '0.1.9', optional = true }
+lefthk-core = { version = '0.2', optional = true }
 leftwm-core = { path = "../leftwm-core", version = '0.4.2' }
 leftwm-macros = {path = "../leftwm-macros", version = '0.4.2'}
 leftwm-layouts = "0.8.4"

